hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -280,6 +280,18 @@ public AzureBlobFileSystemStore(
         "abfs-bounded");
   }
 
+  /**
+   * Updates the client with the namespace information.
+   *
+   * @param tracingContext the tracing context to be used for the operation
+   * @throws AzureBlobFileSystemException if an error occurs while updating the client
+   */
+  public void updateClientWithNamespaceInfo(TracingContext tracingContext)
+      throws AzureBlobFileSystemException {
+    boolean isNamespaceEnabled = getIsNamespaceEnabled(tracingContext);
+    getClient().setIsNamespaceEnabled(isNamespaceEnabled);
```

Comment:
I was wondering if we should update both the clients here itself??
We might be working with both clients in case of fallback scenarios.

Reply:
We are updating the parent with this info, so it will be available to both the clients

Reply:
Ahh didn't noticed its a statci field.
Looks good